libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -264,6 +264,10 @@ - (UIImage*)photo {
 }
 
 - (void)setPhoto:(UIImage *)photo {
+    [self setPhoto:photo completion:nil];
+}
+
+- (void)setPhoto:(UIImage*)photo completion:(void (^ __nullable)(void))completion {
```

Comment:
@jeanbovet Since we are adding a completion block, we should also care about the success/failure. What do you think?

Reply:
That's a great question! I did consider it but backtracked because I wasn't sure how you wanted to treat error. For example, in the current implementation, either error in deleting or storing the image will still set the user: should I change that as well to not assign the photo if storing the photo fails?